tests/it/scargo_test/it_scargo_doc.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pytest
+from typer.testing import CliRunner
+
+from scargo import cli
+
+PRECONDITIONS = [
+    "precondition_regression_tests",
+    "precondition_regular_tests",
+    "precondition_regression_tests_esp32",
+    "precondition_regression_tests_stm32",
+]
+
+
+@pytest.mark.parametrize("precondition", PRECONDITIONS)
+def test_doc(precondition, request):
+    request.getfixturevalue(precondition)
+    doc_html_file = Path("build/doc/html/index.html")
+    runner = CliRunner()
+
+    result = runner.invoke(cli, ["doc"])
+    assert result.exit_code == 0
+    assert doc_html_file.exists()
```

Comment:
is_file() is enough, no need for exists()

Reply:
removed exists()